Schutzfile

```diff
@@ -1,5 +1,10 @@
 {
   "fedora-35": {
+    "dependencies": {
+      "manifest-db": {
+        "commit": "4f596a41565de71bcbec7cd23709cb4888fa7f34"
+      }
+    },
```

Comment:
We should probably make this a global thing, since I don't think it is desirable to have different manifest db versions for different distros. I think that would get very confusing.

Reply:
That's the way it's handled on the other projects. I'm not against changing that though.

Reply:
I went for a `global` section containing the dependency.